lib/vsdata.cpp

```diff
@@ -571,10 +571,11 @@ void KeycPressed(GLenum state)
 
       int precision = prompt<int>("Enter precision (4): ",
                                   &default_precision,
-                                  [](int p){ return p>=0; });
-      char format = prompt<char>("Enter format [(d)efault, (f)ixed, (s)cientific] (d): ",
-                                 &default_format,
-                                 [](char c){ return c=='d' || c=='f' || c=='s'; });
+      [](int p) { return p>=0; });
+      char format =
+         prompt<char>("Enter format [(d)efault, (f)ixed, (s)cientific] (d): ",
+                      &default_format,
+      [](char c) { return c=='d' || c=='f' || c=='s'; });
```

Comment:
Hmm, this looks like an error of astyle 🤔 Maybe try to put it at the end of the previous line?